fuzzers/lafintel/builder.Dockerfile

```diff
@@ -28,6 +28,12 @@ RUN wget https://lcamtuf.coredump.cx/afl/releases/afl-2.26b.tgz -O /afl-2.26b.tg
     tar xvzf /afl-2.26b.tgz -C / && \
     mv /afl-2.26b /afl && \
     cd /afl && \
+    git clone https://github.com/google/AFL.git /afl/recent_afl && \
+    cd /afl/recent_afl && \ 
+    git checkout 8da80951dd7eeeb3e3b5a3bcd36c485045f40274 && \
+    cd /afl/ && \
+    cp /afl/recent_afl/*.c /afl/ && \
```

Comment:
Forgot to ask this: Can we just use the new afl code or what is the need to copy afl files into old afl ?